node/router/messages/Cargo.toml

```diff
@@ -68,3 +68,21 @@ features = [ "codec" ]
 
 [dependencies.tracing]
 version = "0.1"
+
+[dev-dependencies.snarkos-node-sync-locators]
+path = "../../sync/locators"
+version = "=2.1.7"
+features = [ "test" ]
+
+[dev-dependencies.snarkvm]
+workspace = true
+features = [ "algorithms" ]
+
+[dev-dependencies.snarkvm-ledger-test-helpers]
```

Comment:
please use `snarkvm` with `test-helpers` flag

Reply:
just did; however, this doesn't work with the currently pinned snarkVM (it does work with `snarkVM/staging`, though